static/absolute/js/searchPage.js

```diff
@@ -79,6 +80,16 @@ window.onload = function () {
   });
 };
 
+function encodeAddress(address) {
```

Comment:
Why are there 2 encode addresses?

Reply:
Maybe you know a better way to get the address to the server?

To pass the address in the URL to the server, the address still needs to be URL encoded. When the address is taken from the URL params in the server, the encoded address is automatically reverted back to the regular address, so it needs to be encoded again to make the geocode api call. 

I think the URL can sometimes be automatically encoded, but it's not consistent.

Reply:
After changing the structure of the server's geocode api to take URL params instead, I removed the encoding function from the client and pass the address as is.